specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/examples/recommendations/ReevaluateRecommendation.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2024-01-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "recommendationId": "6d4b54eb-8684-4aa3-a156-3aa37b8014bc"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.SecurityInsights/recommendations/6d4b54eb-8684-4aa3-a156-3aa37b8014bc/triggerEvaluation?api-version=2024-01-01-preview"
```

Comment:
we don't return `Azure-AsyncOperation` right?
because we didn't go with async endpoint

Reply:
I believe we do:
![image](https://github.com/Azure/azure-rest-api-specs/assets/19761142/4f609d3c-cbcd-4c16-9728-0ee177cf284e)

Reply:
maybe I missed something, but I make a call to the API and I don't see `Azure-AsyncOperation` in the response headers
![image](https://github.com/Azure/azure-rest-api-specs/assets/49263932/05b77a8a-63c9-43f4-9b07-caf195dd53c2)

Reply:
oh, sorry, didn't see it there
correct, also there we don't have it

Reply:
Sure, removed in both places.

Reply:
See this, I think we have to add it

![image](https://github.com/Azure/azure-rest-api-specs/assets/19761142/fa6ca320-bda9-4b9b-97ef-3fdfe5797b95)

Reply:
Added it back.

Reply:
I see, I think the problem is that we marked it as long running operation, but in fact it doesn't, because we do return immediately and we don't support async operation tracking currently
so we should remove the `x-ms-long-running-operation` tag
![image](https://github.com/Azure/azure-rest-api-specs/assets/49263932/bcdf5941-f4b2-4baa-8b5f-d4f96f481377)

Reply:
Done.